service/src/main/java/uk/gov/hmcts/reform/fpl/enums/Section.java

```diff
@@ -4,15 +4,16 @@
     APPLICANT("applicant", "applicant"),
     CHILDREN("children", "children"),
     ORDERS("orders", "orders and directions needed"),
+    RESPONDENTS("respondent", "respondents"),
     GROUNDS("grounds", "grounds for the application"),
     HEARING("hearing", "hearing needed"),
-    DOCUMENTS("Document", "documents"),
-    CASENAME("caseName", "case name");
+    DOCUMENTS("document", "documents"),
+    CASENAME("casename", "case name");
```

Comment:
Is there a reason why this is no longer camel case? Also should document and respondent be singular or plural?

Reply:
I'm using JSR for validation and it uses the name of the class as a validation key i.e package `uk.gov.hmcts.reform.fpl.model.Proceeding`. Since my new validation class was validating from `RespondentParty` the camel casing would no longer work.

In the second arg in the enum is display text. So it would render "You must add details to case name" etc